### PR TITLE
Add address picker to Create Event page

### PR DESCRIPTION
**Pull request by dwalintukan**
image

image

### CODE REVIEW

**Comment by liviavinci** [comment]
Please add a screenshot for UI changes

**Comment by dwalintukan** [comment]
oops sorry, done

**Comment by liviavinci** [comment]
do you think we are better of with an downward arrow instead of a plus icon on the right of the input?

**Comment by dwalintukan** [comment]
@liviavinci doesn't a down arrow assume that it will be a drop down?

File: src/components/SelectAddressDialog/index.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      dialogVisible,
+      walletAddresses,
+    } = this.props;
+
+    return (
+      <Dialog
+        open={dialogVisible}
+        onClose={this.handleClose}
+      >
+        <DialogTitle>Select Centralized Oracle</DialogTitle>
```
Comment (liviavinci) [inline src/components/SelectAddressDialog/index.jsx]: let's call it result setter and please handle the localization

**Comment by liviavinci** [comment]
Good to go after the above comment and changing the icon

File: src/components/SelectAddressDialog/index.jsx (hunk 1 above)
Comment (dwalintukan) [reply src/components/SelectAddressDialog/index.jsx]: done

**Pull request merged**